tests/build.rs

```diff
@@ -14,7 +14,7 @@ fn it_builds_webpack() {
     let fixture = Fixture::new();
     fixture.scaffold_webpack();
 
-    let wrangler_toml = WranglerToml::webpack_no_config("test-build-webpack");
+    let wrangler_toml = WranglerToml::webpack_zoneless("test-build-webpack", true);
```

Comment:
no_config was kind of a confusing and misleading identifier.

Reply:
i'd separate the `true` out to its own line